demo/src/test/java/io/yupiik/fusion/mcp/demo/test/JsonAsserts.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2025 - present - Yupiik SAS - https://www.yupiik.com
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.yupiik.fusion.mcp.demo.test;
+
+import io.yupiik.fusion.json.internal.JsonMapperImpl;
+import io.yupiik.fusion.json.pretty.PrettyJsonMapper;
+import org.opentest4j.AssertionFailedError;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public final class JsonAsserts {
```

Comment:
can make sense to create a fusion-testing-json module in fusion directly since this pattern is common

could even take an Object as expected value and just prettify its tostring when you want to pass the value as a Map.of() directly (overloading)

Reply:
Done, see https://github.com/yupiik/fusion/commit/f57dbd0f089dd1fbcc8f97ed86b892bf5b3cc0b7